README.md

```diff
@@ -28,16 +43,40 @@ Compatibility matrix:
 Metrics Server | Metrics API group/version | Supported Kubernetes version
 ---------------|---------------------------|-----------------------------
 0.3.x          | `metrics.k8s.io/v1beta1`  | 1.8+
-0.2.x          | `metrics.k8s.io/v1beta1`  | 1.8+
 
 
-In order to deploy metrics-server in your cluster run the following command:
+To deploy Metrics Server in your cluster, run:
 
 ```console
 $ kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.3.6/components.yaml
```

Comment:
If you want, mark this block as shell and remove the `$ `

Reply:
done